MAINTAINERS

```diff
@@ -36,6 +37,11 @@
 	Email = "david.gageot@docker.com"
 	GitHub = "dgageot"
 
+	[people.FrenchBen]
+	Name = "French Ben"
```

Comment:
Looks like you swapped your "first" and "last" name here, compared to the other repo 😉  https://github.com/docker/kitematic/blob/master/MAINTAINERS#L34

Reply:
haha which one is better? 🤔

Reply:
You're real name :innocent:

Reply:
*your* - damn you autocorrect